src/main/target/REVO/target.h

```diff
@@ -203,24 +235,57 @@
 #define USE_UART6
 #define UART6_RX_PIN            PC7
 #define UART6_TX_PIN            PC6
+
+#if defined(ELINF405)
+#define CAMERA_CONTROL_PIN      PB7
```

Comment:
This isn't supported in 4.0

Reply:
OK, I removed the line from the target.h.